### PR TITLE
Clean up of the EventHubsCaptureEventGridDemo sample in the azure-event-hubs repo

### PR DESCRIPTION
**Pull request by zedy-wj**
- The WindTurbineDataGenerator project does not use the old EH package anymore, but it does list it in the [packages.config file](https://github.com/Azure/azure-event-hubs/blob/e9e01dd7b5bb4a32835f5ece855b9d18c579e828/samples/e2e/EventHubsCaptureEventGridDemo/WindTurbineDataGenerator/packages.config#L6) and the [csproj file](https://github.com/Azure/azure-event-hubs/blob/e9e01dd7b5bb4a32835f5ece855b9d18c579e828/samples/e2e/EventHubsCaptureEventGridDemo/WindTurbineDataGenerator/WindTurbineDataGenerator.csproj#L44)
- The FunctionEGDWDumper project lists Azure.Storage.Blobs (the new Storage package) in the [csproj file](https://github.com/Azure/azure-event-hubs/blob/5a16c140921d263b22828fe8b19127f6779465b5/samples/e2e/EventHubsCaptureEventGridDemo/FunctionEGDWDumper/FunctionEGDWDumper.csproj#L9), but uses the older Microsoft.WindowsAzure.Storage package in the [Function.1.cs file](https://github.com/Azure/azure-event-hubs/blob/5a16c140921d263b22828fe8b19127f6779465b5/samples/e2e/EventHubsCaptureEventGridDemo/FunctionEGDWDumper/Function1.cs#L20)
- The DWDumper project lists the older Key Vault package in the csproj file, but does not use it anywhere in code
- The DWDumper project also uses the older Storage package. It should use Azure.Storage.Blobs instead

l have updated these file, and use new package Azure.Storage.Blobs in DWDumper project.

@ramya-rao-a , @jongio for notification.

### CODE REVIEW

**Comment by ramya-rao-a** [comment]
Thanks @zedy-wj !

@serkantkaraca, @spelluru, Please review

**Comment by spelluru** [comment]
@ramya-rao-a - Looks good to me. Thanks.

File: samples/e2e/EventHubsCaptureEventGridDemo/DWDumper/Program.cs
Comment (ramya-rao-a) [inline samples/e2e/EventHubsCaptureEventGridDemo/DWDumper/Program.cs:38]: There are constants that are defined for this purpose a few lines above. Can we not re-use them?
Instead of containerName and blobName, we should be able to use `EventHubsCaptureAvroBlobUri` by creating the BlobClient directly without needing to use the ContainerClient

https://docs.microsoft.com/en-us/dotnet/api/azure.storage.blobs.blobclient.-ctor?view=azure-dotnet

File: samples/e2e/EventHubsCaptureEventGridDemo/FunctionEGDWDumper/Function1.cs
Comment (ramya-rao-a) [inline samples/e2e/EventHubsCaptureEventGridDemo/FunctionEGDWDumper/Function1.cs:71]: Shouldnt we be using `StorageConnectionString` and `fileUri`?

File: samples/e2e/EventHubsCaptureEventGridDemo/DWDumper/DWDumper.csproj
Comment (ramya-rao-a) [inline samples/e2e/EventHubsCaptureEventGridDemo/DWDumper/DWDumper.csproj:42]: The HintPath should not have hard coded values pointing to personal machine

**Review by ramya-rao-a — CHANGES_REQUESTED** [review]

File: samples/e2e/EventHubsCaptureEventGridDemo/DWDumper/DWDumper.csproj
Comment (zedy-wj) [reply samples/e2e/EventHubsCaptureEventGridDemo/DWDumper/DWDumper.csproj:42]: Sorry, it was my mistake!  I used the `Reference Manager` to add reference directly , and forgot to modify it.

File: samples/e2e/EventHubsCaptureEventGridDemo/DWDumper/DWDumper.csproj
Comment (zedy-wj) [reply samples/e2e/EventHubsCaptureEventGridDemo/DWDumper/DWDumper.csproj:45]: `BlobContainerClient` uses the package `Azure.Storage.Common`. l will remove it and use `EventHubsCaptureAvroBlobUri` to create BlobClient.

File: samples/e2e/EventHubsCaptureEventGridDemo/DWDumper/Program.cs
Comment (zedy-wj) [reply samples/e2e/EventHubsCaptureEventGridDemo/DWDumper/Program.cs:38]: I have update it to use `EventHubsCaptureAvroBlobUri`  creating the BlobClient directly.

File: samples/e2e/EventHubsCaptureEventGridDemo/FunctionEGDWDumper/Function1.cs
Comment (zedy-wj) [reply samples/e2e/EventHubsCaptureEventGridDemo/FunctionEGDWDumper/Function1.cs:71]: I missed these defined parameter。I have updated it.

**Comment by zedy-wj** [comment]
@ramya-rao-a  Thanks for your guidance. I have updated according to your requirements. Let me know, if you have any other questions or guidance.

**Comment by ramya-rao-a** [comment]
Thanks @zedy-wj 

@jsquire, @serkantkaraca, Can you review please?

File: samples/e2e/EventHubsCaptureEventGridDemo/DWDumper/DWDumper.csproj
Comment (jsquire) [inline samples/e2e/EventHubsCaptureEventGridDemo/DWDumper/DWDumper.csproj:65]: This reference looks out of place, like it was added with .NET Core rather than a legacy .NET Framework tool.   We should probably continue to use the old format or upgrade everything.

File: samples/e2e/EventHubsCaptureEventGridDemo/WindTurbineDataGenerator/packages.config
Comment (jsquire) [inline samples/e2e/EventHubsCaptureEventGridDemo/WindTurbineDataGenerator/packages.config:4]: We should upgrade this to 5.4.0 while we're here;  no code changes should be needed.

**Review by jsquire — DISMISSED** [review]

File: samples/e2e/EventHubsCaptureEventGridDemo/DWDumper/DWDumper.csproj
Comment (zedy-wj) [reply samples/e2e/EventHubsCaptureEventGridDemo/DWDumper/DWDumper.csproj]: I have updated this reference with the old format.

File: samples/e2e/EventHubsCaptureEventGridDemo/WindTurbineDataGenerator/packages.config
Comment (zedy-wj) [reply samples/e2e/EventHubsCaptureEventGridDemo/WindTurbineDataGenerator/packages.config]: Thanks @jsquire, updated.

**Review by jsquire — APPROVED** [review]

**Comment by ramya-rao-a** [comment]
Thanks @jsquire!

@serkantkaraca, Waiting for your approval

**Review by serkantkaraca — APPROVED** [review]
:shipit:

**Comment by ramya-rao-a** [comment]
Thanks @serkantkaraca!
We don't have merge permissions here. Can either you or @spelluru merge this PR?

**Comment by spelluru** [comment]
@ramya-rao-a - merging it now. Thanks.

**Pull request merged**
